tests/System/integration/administrator/components/com_installer/Languages.cy.js

```diff
@@ -9,8 +9,15 @@ describe('Test in backend that the Installer', () => {
   });
 
   it('has Afrikaans Language installable', () => {
-    cy.get('tr.row0').should('contain.text', 'Afrikaans').then(() => {
-      cy.get('input.btn.btn-primary.btn-sm').should('exist');
+    cy.get('body').then((body) => {
+      if (body.find('#installer-languages table').length === 0) {
+        cy.get('#installer-languages .alert.alert-info').should('contain.text', 'No Matching Results');
+        cy.checkForSystemMessage(`Can't connect to https://update.joomla.org/language/translationlist`);
```

Comment:
```suggestion
        cy.checkForSystemMessage(`Can't connect to https://update.joomla.org/language/translationlist_6.xml`);
```
https://update.joomla.org/language/translationlist ends in a 404

Reply:
It only checks part of the message so that it is also correct for Joomla 7 and following

Reply:
That makes sense.